scss/_buttons.scss

```
@@ -170,7 +170,7 @@
   --#{$prefix}btn-disabled-color: #{$btn-link-disabled-color};
   --#{$prefix}btn-disabled-border-color: transparent;
   --#{$prefix}btn-box-shadow: 0 0 0 #000; // Can't use `none` as keyword negates all values when used with multiple shadows
-  --#{$prefix}btn-focus-shadow-rgb: #{to-rgb(mix(color-contrast($link-color), $link-color, 15%))};
+  --#{$prefix}btn-focus-shadow-rgb: var(--#{$prefix}btn-link-focus-shadow-rgb);
```

```suggestion
  --#{$prefix}btn-focus-shadow-rgb: #{$btn-link-focus-shadow-rgb};
```
To better fit previous comment